### PR TITLE
fix typing

### PR DESCRIPTION
**Pull request by goderbauer**

### CODE REVIEW

**Comment by goderbauer** [comment]
I signed it!

File: dart/sync/lib/src/interfaces.dart [hunk 1]
```diff
@@ -23,7 +23,7 @@ abstract class PageLoader {
   bool get useShadowDom;
   PageLoaderElement get globalContext;
 
-  Object getInstance(Type type, [dynamic context]);
```
Comment (DrMarcII) [inline dart/sync/lib/src/interfaces.dart]: Why the change? Although not directly addressed, I believe that https://www.dartlang.org/effective-dart/design/#do-annotate-with-object-instead-of-dynamic-to-indicate-any-object-is-accepted is the closest to a recommendation for this case.

Comment (goderbauer) [inline dart/sync/lib/src/interfaces.dart]: Sorry, I am closing this PR. It is incomplete.
